src/unibasicTemplates/getServerInfo.njk

```diff
@@ -44,6 +44,44 @@ if udoSetProperty(output, 'delimiters', delimiters) then
   go returnFromSub
 end
 
+if udoCreate(UDO_OBJECT, limits) then
+  call error_handler(ERROR_UDO, output)
+  go returnFromSub
+end
+
+hush off
```

Comment:
"hush", which is enabled in the entry point, had to be disabled in order to get the `CAPTURING` clause to work here.